docs/ruleset_functions/get_zone_peak_internal_load_floor_area.md

```diff
@@ -0,0 +1,48 @@
+# get_zone_peak_internal_load_floor_area
+**Schema Version:** 0.0.21
+**Description:** finds the peak coincident internal loads of a zone and returns the value in btu/h/ft2
+
+**Inputs:** 
+- **RMR**
+- **zone**
+
+**Returns:**  
+- **result**: an array giving 2 values: [total peak btu/h/ft2 in the zone, total zone area] the total peak btu/sf is the internal coincident peak loads in all spaces in the zone
+ 
+**Function Call:**
+
+
+## Logic:
+
+- create list of all loads in the zone.  Each load is an array of: [the peak in btu/h/ft2, schedule hourly values,schedule_multiplier], the peak in btu/h/ft2 takes into account the sensible and latent fraction, for where there are multiple internal_loads in a space, the schedule multiplier indicates the % of total of the internal load: `loads = []`
+- create an area variable which will be the sum of all space areas: `zone_area = 0`
+- For each space in zone: `for space in zone.spaces:`
+	- add the space.floor_area to the area variable: `zone_area += space.floor_area`
+	- create lights power variable for storing the total space lighting power: `lights_power = 0`
+	- For each lighting object in the space: `for lights in space.interior_lighting:`
+		- add the space lighting power to the lights_power variable & convert from W/m2 to btu/hr/ft2: `lights_power += lights.power_per_area * 0.316998331`
```

Comment:
lights.power_per_area should be multiplied by space.floor_area to get light_power.

Also, I don't think we should include unit conversions here and in other places below because RCT converts from metric to IP when it parces the RMD file.

Reply:
changed